content/updates/quality/1-The-quality-score.md

```diff
@@ -0,0 +1,32 @@
+# Introducing our Quality score!
+
+## A new way of scoring...
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
+:::
+Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for better clarity.
+
+![The new quality score](../img/quality-score.png)
+
+You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.
+
+![The new quality score in the product edit form](../img/quality-score-in-pef.png)
+
+## ...All your products...
+::: meta-data type="New" ee-only="false"
+:::
+You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! You will get specific quality scores for all your variant products so you can quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level so you can see at a glance on which level the attributes need to be improved (at the variant product or at the product model level).
+
+![DQI on variant products](../img/dqi-for-a-variant-product.png)
+
+When you have improvement recommendations, you can simply click on your attribute to go directly to it and improve it under the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
```

Comment:
```suggestion
When you have improvement recommendations, you can simply click on your attribute and improve it right from the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
```